xml/System.Security.Cryptography.Pkcs/SubjectIdentifier.xml

```diff
@@ -52,10 +52,19 @@
         <Parameter Name="certificate" Type="System.Security.Cryptography.X509Certificates.X509Certificate2" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
       </Parameters>
       <Docs>
-        <param name="certificate">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="certificate">The certificate to verify for a match.</param>
```

Comment:
This sentence sounds really non-intuitive IMO but feels there should be `Subject identifier` somewhere in the description (not sure how to word it pretty)

Reply:
Can you provide a suggestion? We can reword it with Ron and Maira's help if necessary.

I don't know this API well enough to understand what you'd like to see.

Reply:
How about:

```suggestion
        <param name="certificate">The certificate to match with the current subject identifier instance.</param>
```

Reply:
LGTM